src/main/drivers/pwm_output.h

```diff
@@ -20,7 +20,9 @@
 #include "drivers/io_types.h"
 #include "timer.h"
 
+#ifndef MAX_SUPPORTED_MOTORS 
 #define MAX_SUPPORTED_MOTORS 12
```

Comment:
For readability / consistency's sake, would it make sense to stick this into `common_fc_pre.h` too?